logstash-core/lib/logstash/environment.rb

```diff
@@ -1,7 +1,27 @@
 # encoding: utf-8
 require "logstash/errors"
+require "logstash/config/cpu_core_strategy"
 
 module LogStash
+
+  DEFAULT_SETTINGS = {
+    "node.name" => Socket.gethostname,
```

Comment:
would it be good to make this key strings also constants somewhere? what do you think?